src/icons.rs

```diff
@@ -26,8 +26,7 @@ lazy_static! {
         "volume_full" => " VOL ",
         "volume_half" => " VOL ",
         "volume_empty" => " VOL ",
-        // This icon has no spaces around it because it is manually set as text. (sound.rs)
-        "volume_muted" => "MUTED",
+        "volume_muted" => " VOL MUTED ",
```

Comment:
Was VOL added just for consistency?

Reply:
It was to keep the output consistent with the previous output.

Reply:
Ah, I see what's going on now. The output is still the same as before.